.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!-- Thank you for contributing to our WordPress open source project! -->
+
+## Description
+<!-- Please provide a clear and concise description of your changes -->
+<!-- What problem does this PR solve? What functionality does it add/improve? -->
+
+## Related Issue
+<!-- Please link any related issues here. Use the syntax: "Fixes #123" or "Related to #456" -->
+
+## Dependant PRs
+<!-- List any dependent PR's that are awaiting review. Use the following syntax: "#1234" -->
+
+## Type of Change
+<!-- What types of changes does your code introduce? Put an x in all boxes that apply -->
+- [ ] ✅ Bug fix (non-breaking change which fixes an issue)
+- [ ] ✨ New feature (non-breaking change which adds functionality)
+- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] 🧹 Code refactoring (no functional changes)
+- [ ] 📄 Example update (no functional changes)
+- [ ] 📝 Documentation update
+- [ ] 🔍 Performance improvement
+- [ ] 🧪 Test update
+
+## How Has This Been Tested?
+<!-- Please describe the tests you've run to verify your changes -->
+<!-- Include details of your testing environment, and the tests you ran -->
+
+## Screenshots
+<!-- If your change affects the UI or UX, provide before/after screenshots or videos -->
+<!-- Delete this section if not applicable -->
+
+## Checklist
```

Comment:
not sure how I feel about the checklist. also eels like the kinda things that gets ignored. but I do like it.